rosidl_generator_py/CMakeLists.txt

```diff
@@ -27,6 +27,10 @@ if(BUILD_TESTING)
   include(cmake/register_py.cmake)
   include(cmake/rosidl_generator_py_get_typesupports.cmake)
 
+  include_directories(include
+    ${rosidl_runtime_c_INCLUDE_DIRS}
```

Comment:
It would be nice to only do this on the targets that need it. Using either `ament_target_dependencies()` or `target_include_directories()`.

Reply:
Any why is this necessary now? The same include wasn't present for the generator before.

Reply:
removed